### ISSUE TITLE
[Feature Request] Add the ability to set viewing permissions to the individual user/player

### ISSUE THREAD

**Issue by p4535992** [mouse0270/kasper#20]
At the moment you can only set display or non-display for all players .
To keep certain factions secret I would need to be able to set the display on the individual user.

Something like an "Ownership Configuration" item on the configuration panel.

If it were possible with ui-level references of the ownership to avoid opening configuration windows every time.

**Comment by mouse0270** [comment]
So to clarify, you'd like the dialog to allow you to select individual users instead of just a toggle to show to all or not? Similar to the Ownership Configuration dialog like this:
![image](https://github.com/mouse0270/kasper/assets/564874/399b8efe-7e03-431d-9876-96be565c2f37)


> If it were possible with ui-level references of the ownership to avoid opening configuration windows every time.
I believe your saying, if the user has observer or ownership level permissions of the linked document they should be able to see it? 
- If this is what you are asking to, the reason this currently is not done, is because you may be using KASPER to track secret values for the player you may not want them to see. For example, I ran a horror campaign a while back in which the players would make wisdom checks, and if they failed, I would move a marker down, if they succeeded I would move the marker up. This type of data I would not want the user to see as it adjust the players DC based on success/fails
- If this is not what your asking, can you please clarify?

**Comment by p4535992** [comment]
A ok if the module is only meant to make the GM keep track then that's fine.
It would have suited me to have the "Thieves Guild" reputation only be seen by the thief in the group for example, but if it's something outside the purpose of the module then it doesn't matter

**Comment by mouse0270** [comment]
Oh I can totally make it so that you can give individual players instead of everyone. I will add the feature to allow you to share it with specific players instead of everyone.

I just don't want it to be based off if the player has ownership or observer rights.

**Comment by p4535992** [comment]
Yes a limitation for user/player  (not the ownership) is more than enough for my use case
